skipruntime-ts/src/Extern.sk

```diff
@@ -448,10 +450,43 @@ fun toSkdb(
     context,
     mapHandle.toRow,
     connected != 0,
-    asKeyRanges(rangeOpt),
+    asKeyRangesOpt(rangeOpt),
   )
 }
 
+fun identityMap<K: Key, V: File>(
+  _context: mutable Context,
+  writer: mutable TWriter<K, V>,
+  key: K,
+  it: mutable NonEmptyIterator<V>,
+): void {
+  writer.setArray(key, it.collect(Array));
+}
+
+@export("SkipRuntime_sliced")
+fun sliced(
```

Comment:
Sorry to come late and nitpick names, but the other operation names like map and reduce are all verbs, while this one is a noun...